### ISSUE TITLE
['Win10', 'Py3.72', 'pipx-0.12.3.1'] TypeError: unsupported operand type(s) for +=: 'set' and 'list'

### ISSUE THREAD

**Issue by Laurentiu-Andronache** [pipxproject/pipx#96]
Wait! Don't help me! I'm trying to submit a commit and fix this one myself!

```
C:\Users\Nebu>pipx uninstall-all
Traceback (most recent call last):
  File "C:\Python37\Scripts\pipx-script.py", line 11, in <module>
    load_entry_point('pipx==0.12.3.1', 'console_scripts', 'pipx')()
  File "c:\python37\lib\site-packages\pipx\main.py", line 496, in cli
    exit(run_pipx_command(parsed_pipx_args, binary_args))
  File "c:\python37\lib\site-packages\pipx\main.py", line 191, in run_pipx_command
    commands.uninstall_all(PIPX_LOCAL_VENVS, LOCAL_BIN_DIR, verbose)
  File "c:\python37\lib\site-packages\pipx\commands.py", line 435, in uninstall_all
    uninstall(venv_dir, package, local_bin_dir, verbose)
  File "c:\python37\lib\site-packages\pipx\commands.py", line 415, in uninstall
    binary_paths += dep_paths
TypeError: unsupported operand type(s) for +=: 'set' and 'list'
```

**Comment by Laurentiu-Andronache** [comment]
relevant pull request https://github.com/pipxproject/pipx/pull/97

**Comment by cs01** [comment]
This has been fixed in 0.12.3.2
